js/components/arrowButtonMaker.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Creates a button of a specific type.
+ * @param {object} props - The properties of the arrow button.
+ * @param {string} props.type - The type of button ("left" or "right").
+ * @param {int} props.index - We're passing the current fighter step index for onClick to work.
+ * @returns {HTMLElement} - The generated button element.
+ * @throws {Error} - Throws an error if the type is not "left" or "right".
+ */
+
+import { chooseFighterStep } from '../wizard/steps/choose-fighter-step/choose-fighter-step.js';
+
+export function arrowButtonMaker(props) {
+   const type = props.type;
+   const index = props.index;
+
+   if (type !== 'left' && type !== 'right') {
```

Comment:
**nitpick:** Here is the same problem like below. Why you limit yourself to specific use case instead of passing backgroundImage as prop. You can remove this `if` statement, as it's dirty, and non maintainable. Would be much more simple, and clean to do the following:

```
 const type = props.type;
   const index = props.index;
 const backgroundImage = props. backgroundImageSrc;
```